src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -334,21 +330,27 @@ Git: 1.4.2-37-g9da9b504fc
       <feature bit="8" code="PRES" name="Presets" summary="Thermostat supports setpoint presets">
         <optionalConform/>
       </feature>
-      <feature bit="9" code="TEVT" name="Events" summary="Thermostat supports events" apiMaturity="provisional">
+      <feature bit="9" code="TEVT" name="Events" summary="Thermostat supports events">
```

Comment:
Shouldn't this maintain the apiMaturity="provisional" part?

Reply:
yes i need to regenerate it with apiMaturity set to loose